### PR TITLE
Fix release automation to update values.yaml correctly in release-1.0

### PR DESCRIPTION
**Pull request by rkpagadala**
change substitution to work with any branch

### CODE REVIEW

File: release/airflow/modify_values.sh [hunk 1]
```diff
@@ -26,30 +26,32 @@ function fix_values_yaml_worker() {
...
+  hub="$6"
 
   gsutil -q cp "${gcs_folder_path}/${tarball_name}" .
   eval    "$unzip_cmd"     "${tarball_name}"
   rm                       "${tarball_name}"
 
-  sed -i "s|hub: gcr.io/istio-release|hub: ${HUB}|g" ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
-  sed -i "s|tag: master-latest-daily|tag: ${TAG}|g"  ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
+  sed -i "s|hub: gcr.io/istio-release|hub: ${hub}|g" ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
+  sed -i "s|tag: .*-latest-daily|tag: ${TAG}|g"  ./"${folder_name}"/install/kubernetes/helm/istio*/values.yaml
```
Comment (hklai) [inline release/airflow/modify_values.sh]: Should we use ${branch} instead of .*?

Comment (rkpagadala) [reply release/airflow/modify_values.sh]: no, that makes it brittle and subject to changes in the master/release branch. Since it is being replaced with the correct tag this is a better fix.

File: release/pipeline/dags/istio_common_dag.py [hunk 2]
```diff
@@ -89,6 +89,7 @@ def getBashSettingsTemplate(extra_param_lst=[]):
   for key in keys:
     template_list.append("export %s={{ settings.%s }}" % (key, key))
   template_list.append("""
+                gsutil -q cp gs://istio-release-pipeline-data/release-tools/test-version/data/release/* .
```
Comment (hklai) [inline release/pipeline/dags/istio_common_dag.py]: why is this needed now?

**Comment by rkpagadala** [comment]
@hklai PTAL
